.pre-commit-config.yaml

```diff
@@ -23,7 +23,7 @@ repos:
     rev: 'v0.0.292'
     hooks:
     - id: ruff
-      args: [--fix]
+      args: [--fix, --ignore, PLR0913]
```

Comment:
This should go in the `pyproject.toml` `per-file-ignores` block: https://github.com/IMAP-Science-Operations-Center/imap_processing/blob/10e75630bdca2637a8744356ffe3d873f7711d2b/pyproject.toml#L73

Reply:
Thank you! I was wondering and couldn't figure out. Could I add to this list instead with others.

```
ignore = ["D100", "D104", "PLR2004", "S101", "PLR0913"]
```

Reply:
I'd prefer not because then that applies to every file and I think it is good to leave in as a general check.

Reply:
ok. Will add to per-files-ignores